pcdsdevices/spectrometer.py

```diff
@@ -218,6 +218,32 @@ def __init__(self, prefix, *, name, prefix_focus, prefix_energy, **kwargs):
                          prefix_energy=prefix_energy, **kwargs)
 
 
+class VonHamosCrystal_2(BaseInterface, GroupDevice):
+    """Translation, rotation and tilt motors for control of a single crystal."""
```

Comment:
This is the only name that's kind of wonky- an unexplained _2.
I think either the name or the docstrings needs to explain when to use it.

The rest of the classes here have similar names but hopefully they have descriptive enough docstrings...

Reply:
This is probably to differentiate this from an existing VonHamosCrystal.  https://github.com/pcdshub/pcdsdevices/blob/781df8b23e111916e9f052fb43dd3a39ca5f6cd9/pcdsdevices/spectrometer.py#L109-L115

This seems to just be a different coordinate system.   They effectively do the same thing right?  I'm not volunteering, but this could be some fancy autodetecting pseudopositioner class that accepts either set of PVs?

(not for this PR of course)

Reply:
Yeah I am not too happy with that name either. Another way would be to define the single crystal class inside the main spectrometer class, and then the names wouldn't be a probem. Not sure that's much more elegant though... Thoughts?

Reply:
For now I think just more descriptive names are sufficient.  Maybe:
- VonHamosCrystal_YPT  (Yaw-pitch-trans)
- VonHamosCrystal_Spherical (this is basically a spherical coordinate system, right?)

Reply:
It's not quite spherical I think...
Also I doubt that we'll ever get to re-use the same crystal class for another spectrometer, so the more I think about the it, the more I kinda like the idea of embedding the crystal class inside the spectrometer class. But I don't feel very strongly about any names here...

Reply:
I don't mind if it doesn't get renamed, but the description should say "this is for the mfx spectrometer", for example.